Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -69,16 +74,17 @@ public sealed class BlockStoreLoop
         }
 
         /// <summary>
-        /// 
         /// Initialize the BlockStore
-        ///                 
+        /// <para>
         /// If StoreTip is null, the store is out of sync.
+        /// 
         /// This can happen when:
         ///     1: The node crashed
         ///     2: The node was not closed down properly
         ///     
```

Comment:
XMLDOC.FORMAT rule - use `<list>` and `<item>` here, but note that those are not well formatted in MSVS, so it is handy to use something like 

```
         /// <para>This can happen when:</para>
         /// <list>
         ///     <item>the node crashed;</item>
         ///     <item>the node was not closed down properly.</item>
         /// </list>   
```
or
```
         /// <para>This can happen when:</para>
         /// <list>
         ///     <item>1. The node crashed.</item>
         ///     <item>2. The node was not closed down properly.</item>
         /// </list>   
```

I.e. to use correct punctuation that will act as delimiters when it is not correctly formatted - so that the product is actually readable in MSVS IntelliSense.